egs/sprakbanken/s5/local/chain/run_tdnn.sh

```diff
@@ -0,0 +1 @@
+tuning/run_tdnn_1b.sh
```

Comment:
Also, you have `run_tdnn_lstm.sh` and `run_tdnn.sh`, but you're missing `local/chain/run_lstm.sh` Any reason why you don't have the last file?